src/lib/components/Card.svelte

```diff
@@ -112,6 +114,10 @@
       padding: 0;
     }
 
+    &.noMarginBottom {
+      margin-bottom: 0;
```

Comment:
Do I see correctly on line 180 that if the card has an `href`, then it already has no margin?
Any idea why those 2 things are related?
Is it possible that we can just remove the margin in all cases rather than adding more and more special cases?

Reply:
Here we remove only a bottom margin. The top margin is still needed. Not sure it worth the risk, the card is so generic and widely used component.

Reply:
The possibility to add an `href` to the card and have it be an `<a>` instead of an `<article>` as a result was only added last August (in [this PR](https://github.com/dfinity/gix-components/pull/262)) and removing the margin for those cards was added on the same day (in [this PR](https://github.com/dfinity/gix-components/pull/267)).

I think it's pretty bad to have the styling change based on whether there is a link. And I think it's worse to have multiple ways to remove different parts of the margin. I'm afraid will end up with 4 different props for all different margins.

Instead of having a prop for just the bottom margin, can we have a prop that removes all the margins? If we need a top margin, we can add it back in nss-dapp.

Ideally we would then remove the special styling for `<a>` cards and apply the same prop to get the same effect where needed. But we can think about that separately.

Reply:
This sounds as a lot of refactoring. Let's do it separately. Good investigation btw 👍

Reply:
It's not a lot of refactoring. All I'm asking is that you change `noMarginBottom` to `noMargin` and that it removes all the margins instead of just the bottom margin.

Reply:
Done, thanks.